Bio/motifs/minimal.py

```diff
@@ -17,7 +17,6 @@ def read(handle):
     """Parse the text output of the MEME program into a meme.Record object.
 
     Example:
-
```

Comment:
Does this (and the other case in this file) also need an RST tweak?

Reply:
Yep